### PR TITLE
Removes Skeleton from the Learn page and replaces the flex implementa…

### PR DESCRIPTION
**Pull request by Pjb518**
…tion in the footer with Tachyon styles. Adds custom CSS code which affect the global font size, as well as letter spacing and font weight for the section and page headings sitewide.

Pertains to this issue: https://github.com/rust-lang/www.rust-lang.org/issues/780

### CODE REVIEW

**Comment by skade** [comment]
There's two tiny issues on 1024x768

Screenshot_2019-05-20 Learn - Rust programming language(1)
 (iPad resolution if you use the Firefox mobile view simulator).

The horizontal lines and the buttons are not aligned.

Screenshot_2019-05-20 Learn - Rust programming language(2)

The buttons and the text are not aligned here.

File: src/main.rs [hunk 1]
```diff
@@ -280,7 +280,7 @@ fn concat_vendor_css(files: Vec<&str>) {
 fn main() {
     compile_sass("app");
     compile_sass("fonts");
-    concat_vendor_css(vec!["skeleton", "tachyons"]);
+    concat_vendor_css(vec!["tachyons"]);
```
Comment (skade) [inline src/main.rs]: Please remove this change from the patchset, as other pages are not fixed yet :).

Comment (Pjb518) [reply src/main.rs]: _faceplam_

I made a specific mental note about this when I started working on the file and then pushed it all the same lol

**Comment by Pjb518** [comment]
Looks like the alignment issue exists in Chrome as well, and is also present in the current live site.

Firefox Live site:
Screenshot from 2019-05-20 18-10-01

Chrome Live site:
Screenshot from 2019-05-20 18-11-38

I'll submit a fix shortly.

**Comment by skade** [comment]
@Pjb518 This looks quite good now. Am I overlooking anything or is this ready to merge?

**Comment by pietroalbini** [comment]
An existing inconsistency of that page is the padding the first and last columns have.

<details><summary>Screenshot</summary>



</details>

In all the other Tachyons pages that extra margin is not present. This is also visible in any other page's footer:

<details><summary>Screenshot</summary>



</details>

**Comment by skade** [comment]
Merging this to allow people to depend on the header and footer fixes. Outstanding issues tracked as #786

**Pull request merged**